app/Enemies/enemy.gd

```diff
@@ -0,0 +1,23 @@
+extends RigidBody2D
+
+
+# Declare member variables here. Examples:
+# var a = 2
+# var b = "text"
+
+
+# Called when the node enters the scene tree for the first time.
+func _ready():
+	position = Vector2(300, 50)
+
+
+
+# Called every frame. 'delta' is the elapsed time since the previous frame.
+#func _process(delta):
+#	pass
+
+func _on_RigidBody2D_body_entered() -> void:
+	print("Enemy detects collision")
+
+func _on_RigidBody2D_body_shape_entered():
+	print("Enemy detects shape collision")
```

Comment:
we were using separate folders for scripts and scenes, do you want to discuss changing the folder structure? I could be convinced but we haven't really discussed it and this is creating a lot of line changes